<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingCart</class>
 <widget class="QWidget" name="ShoppingCart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>-1</pointsize>
    <bold>false</bold>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>购物车</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局背景色 */
QWidget {
    background-color: #f5f5f5;
    font-family: &quot;Microsoft YaHei&quot;, Arial, sans-serif;
    font-size: 20px;
}

/* 标题样式 */
QLabel#Title {
    font-size: 22px;
    color: #333333;
    font-weight: bold;
    margin-bottom: 20px;
}
/* 标签行样式 */
QLabel#headerLabel {
    font-size: 20px;
    color: #555555;
    font-weight: bold;
    padding: 6px 0;
    border-bottom: 1px solid #dddddd;
    background-color: #f9f9f9;
    text-align: center;
}

/* 滚动区域样式 */
QScrollArea {
    border: 1px solid #dddddd;
    background-color: #ffffff;
    border-radius: 8px;
}

QScrollArea QWidget {
    background-color: #ffffff;
}

/* 图片 LOGO 样式 */
QLabel#clientpingCartLogo {
    border: 1px solid #dddddd;
    border-radius: 8px;
    margin: 10px;
}

/* 已选商品数量和总价字体样式 */
QLabel#SelectedNumber, QLabel#Total {
    font-size: 20px;
    color: #555555;
}

QLabel#selectedValue, QLabel#TotalValue {
    font-size: 20px;
    color: #0078D7;
    font-weight: bold;
}

/* 按钮样式 */
QPushButton#Settle {
    background-color: #0078D7;
    color: white;
    border-radius: 8px;
    padding: 10px;
    font-weight: bold;
}

QPushButton#Settle:hover {
    background-color: #005EA6;
    border: 1px solid #004A8C;
}

QPushButton#Settle:pressed {
    background-color: #004A8C;
}

/* 全选按钮样式 */
QRadioButton#checkAll {
    font-size: 20px;
    color: #555555;
}

/* 鼠标悬停效果 */
QLabel:hover {
    color: #0078D7;
    text-decoration: underline;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QVBoxLayout" name="leftPanelLayout">
     <item>
      <layout class="QHBoxLayout" name="headerLayout">
       <item>
        <widget class="QLabel" name="headerLabel_5">
         <property name="text">
          <string>商品图片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="headerLabel_4">
         <property name="text">
          <string>商品名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="headerLabel_3">
         <property name="text">
          <string>商品类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="headerLabel_2">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>-1</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>购买数量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="headerLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>商品价格</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1670</width>
          <height>930</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="cartItemLayout"/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightPanelLayout">
     <item>
      <widget class="QLabel" name="Title">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>购物车</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="clientpingCartLogo">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/image/9.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="selectedLayout">
       <item>
        <widget class="QLabel" name="SelectedNumber">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>已选商品：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectedValue">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="totalLayout">
       <item>
        <widget class="QLabel" name="Total">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>总价：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TotalValue">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>¥0.00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="checkAll">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Settle">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>去结算</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
